arches_her/media/js/views/components/reports/digital-object.js

```diff
@@ -64,7 +64,8 @@ define([
                             data: [{
                                 key: 'File',
                                 value: self.getFileName(self.getRawNodeValue(self.resource(), 'file content', 'file')),
-                                type: 'kv',
+                                href: self.getNodeValue(self.resource(), 'file content', 'file'),
```

Comment:
This seems to be creating a bad link address. In this example the link should be *http://localhost:8002/files/d0e52338-cd3c-11ec-872b-0242ac130007* but it is appearing as *http://localhost:8002/files/d0e52338-cd3c-11ec-872b-0242ac130007%20%7C%20/files/d0e9e4ae-cd3c-11ec-872b-0242ac130007*

![image](https://user-images.githubusercontent.com/25295526/168991688-ad853092-c60a-4d4f-99d2-dd574cde9029.png)

Reply:
... and what I can see from the Dev Tools...
![image](https://user-images.githubusercontent.com/25295526/168992218-561f5b2e-5f4e-45d0-807d-90a0f50656c8.png)